bindings/java/iota-wallet-java/lib/native/src/lib.rs

```diff
@@ -47,6 +69,11 @@ pub extern "system" fn Java_org_iota_api_NativeApi_sendMessage(
     _class: JClass,
     command: JString,
 ) -> jstring {
+
+    if env.exception_check().unwrap() {
```

Comment:
Safe to unwrap?

Reply:
I think this should be good.
Since if the JVM fails then failing in Rust at this point makes sense too I would say.

Reply:
In this case returning something to the JVM doesn't make sense if I understood it correctly.